packages/kit-bg/src/providers/ProviderApiTon.ts

```
@@ -5,7 +5,7 @@ import {
 } from '@onekeyfe/cross-inpage-provider-errors';
 import { IInjectedProviderNames } from '@onekeyfe/cross-inpage-provider-types';
 import BigNumber from 'bignumber.js';
-import { isNaN } from 'lodash';
+import { get, isEmpty, isNaN } from 'lodash';
```

Biome (2.1.2)

[error] Do not shadow the global "isNaN" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)